data/pages-content/cfp.md

```diff
@@ -0,0 +1,72 @@
+---
+title: CFP
+subtitle: EuroPython Call for Proposals
+---
+# Call for Proposals
+
+🐍 Pythonistas, mark the dates:
+
+<div style={{textAlign: "center"}}>
+**EuroPython 2023 Call for Proposals (CFP) will be open** between
+
+ **Monday, 6 March 2023** and
+
+ **Sunday, 19 March 2023** [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
+</div>
+
+
+
+We invite proposals from the whole community on a wide range of topics. We welcome speakers at all python levels with any experience with public speaking.
+
+## The Conference Format ##
+
+The conference will feature around 16 Tutorials/Workshops and 120 talk sessions:
+
+* **Tutorials/Workshops (17-18 July)**: hands-on **180-mins sessions**
+
+* **Talks (19-21 July)** : **30, 45, or 60-minute** presentations of Python general topics & PyData related topics
+
+All presentation content can be at **beginner, intermediate or advanced** level.
+
+[comment]: # (Other than talks and tutorials, you are also welcome to submit **posters**: printed in large formats, it is a graphical way to showcase a project or a technology. Posters are exhibited at the conference, read at any time by participants, and can be discussed face to face with their authors during the poster session.)
+
+
+## Topics ##
+
+We welcome presentations focusing on a variety of topics all around Python, e.g. programming, engineering, cloud, DevOps, web, Django, hardware, community, education, or whatever you deem suitable - **it is your conference!**
+
+To see the type of topics presented at previous EuroPython events, check out our [past EuroPython conferences](https://www.europython-society.org/europython/).
+Browse through our past [talk videos](https://www.youtube.com/c/EuroPythonConference) for more inspiration.
+
+For PyData related ideas, take a look at PyData events: [pydata.org](https://pydata.org/).
```

Comment:
Do we need this? If we do, then we should also add links to web/Django, maker-y, testing, cultural, etc etc etc ideas.

Reply:
Since programme team always push for pydata tracks, I'll soften it a bit and go with 

> For ideas to contribute to the PyData track, take a look at PyData events

`